### PR TITLE
reimport v1 entries if they were not fetched

### PR DESCRIPTION
**Pull request by tcitworld**
If an article was not properly fetched on v1, this will reimport it.

About tests, does adding a untitled entry in wallabag-v1.json is enough ?

### CODE REVIEW

File: src/Wallabag/ImportBundle/Import/WallabagV1Import.php [hunk 1]
```diff
@@ -123,6 +126,10 @@ public function setFilepath($filepath)
     protected function parseEntries($entries)
     {
         $i = 1;
+        /*
+         * Untitled in all languages from v1. This should never have been translated
+         */
```
Comment (j0k3r) [inline src/Wallabag/ImportBundle/Import/WallabagV1Import.php]: You can put the comment on one line starting with `//`

File: src/Wallabag/ImportBundle/Import/WallabagV1Import.php [hunk 2]
```diff
@@ -123,6 +126,10 @@ public function setFilepath($filepath)
...
+        $untitled = array('Untitled', 'Sans titre', 'podle nadpisu', 'Sin título', 'با عنوان', 'per titolo', 'Sem título', 'Без названия', 'po naslovu', 'Без назви');
```
Comment (j0k3r) [inline src/Wallabag/ImportBundle/Import/WallabagV1Import.php]: Since the v2 import will also get that, can't you also include an empty string ?

**Comment by j0k3r** [comment]
> About tests, does adding a untitled entry in wallabag-v1.json is enough ?

You can add both for the v1 & v2 import file

File: src/Wallabag/ImportBundle/Import/WallabagV1Import.php (hunk 2 above)
Comment (tcitworld) [inline src/Wallabag/ImportBundle/Import/WallabagV1Import.php]: Don't you mean `No title found` ? In regard to https://github.com/j0k3r/graby/pull/42 ?

Comment (j0k3r) [inline src/Wallabag/ImportBundle/Import/WallabagV1Import.php]: Maybe both.

**Comment by j0k3r** [comment]
:+1:

**Pull request merged**